SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -235,8 +215,28 @@ void ActorSystem::Advance()
 		}
 	}
 
-	CreateEntityHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
-	ClaimPartitionHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
+	for (const EntityDelta& Delta : TombstoneSubView->GetViewDelta().EntityDeltas)
```

Comment:
Was there a reason this was moved till after the other subview processing?

Reply:
yeah, so the main subview can process removals before the tombstone system has its way with them

this might be overcautious, but would rather treat actors that were previously checked out as removals of the main subview than new actors which are tombstoned